confd/templates/functional-tests/kilda.properties.tmpl

```diff
@@ -0,0 +1,100 @@
+# Generated by confd.
```

Comment:
is it really generated file?
I thought the `properties.template` file is predefined, and the real `properties` file is generated

Reply:
This is added in all template files, so I'm not changing this now. I guess the idea was that this info also gets copied to the actual generated properties file, where it is actually generated by confd

Reply:
it makes sense, thx.